sdk/iot/hub/samples/README.md

```diff
@@ -0,0 +1,76 @@
+---
+page_type: sample
+languages:
+  - c
+products:
+  - azure
+  - azure-iot
+  - azure-iot-hub
+urlFragment: iot-hub-samples
+---
```

Comment:
Remove this preamble? Is this meant to act as hidden metadata? It affects readability of the markdown on GitHub:
![image](https://user-images.githubusercontent.com/6527137/80770489-4fe92c00-8b05-11ea-91a9-725954fc928d.png)

Reply:
odd. I think it is this way across all of our stuff. I will likely leave it here for now but make a note to fix across the board if necessary.